ui/nuxeo-data-table/data-table-checkbox.js

```diff
@@ -29,13 +29,21 @@ import '../widgets/nuxeo-checkmark.js';
           :host(:focus) {
             outline: none;
           }
-
+          /**
+            @deprecated since 3.0.5 we no longer use the header property to control the checkbox visibility 
```

Comment:
Better bump this again for 3.0.6 I think. (check before merging)

Reply:
Yup, sadly it didn't make it for 3.0.5 😞